third_party/2.7/dateutil/relativedelta.pyi

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Union
+from datetime import date, datetime, timedelta
+
+__all__ = ...  # type: List[str]
+
+
+class weekday(object):
+    weekday = ... # type: int
+    n = ... # type: int
+    def __init__(self, weekday: int, n=...: Optional[int]) -> None: ...
```

Comment:
I think you meant `n: Optional[int] = ...`.
(Type annotations and default arguments are order-sensitive.)